models/comments-model.js

```diff
@@ -0,0 +1,18 @@
+const connection = require('../db/connection');
+
+const selectComments = (article_id) => {
+    return connection.query(`SELECT * FROM comments
+                            WHERE article_id = $1
+                            ORDER BY created_at DESC`, [article_id])
+                            .then(({rows}) => {
+                                if(rows.length === 0){ 
```

Comment:
This will return 404 whether we have asked for an article that doesn't exist yet (e.g. 9999) OR if we ask for a valid article (e.g. 2) that does not have comments. We need to find a way to check if we have an existing article first...